docs/modules/ROOT/pages/jakartaee-certification/5.21.2/tck-results-full-5.21.2.adoc

```diff
@@ -0,0 +1,738 @@
+[[tck-results]]
+= TCK Results
+
+As required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for releases of Jakarta EE Platform, Full Profile.
+
+**Payara Server 5.21.2 Full Profile Certification Summary**
+
+- Product Name, Version and download URL (if applicable):
++
+https://payara.fish/downloads[Payara Server 5.21.2]
+
+- Specification Name, Version and download URL:
++
+https://jakarta.ee/specifications/platform/8/[Jakarta EE Platform, Full Profile 8]
+- TCK Version, digital SHA-256 fingerprint and download URL:
++
+http://download.eclipse.org/jakartaee/platform/8/eclipse-jakartaeetck-8.0.2.zip[Jakarta EE Platform CTS 8.0.2]
+, SHA-256: `14a21b617bb646055c2952f1422ec04a71389fb37301e1c2969f6c3700aee965`
+
+- Public URL of TCK Results Summary:
++
+http://docs.payara.fish/v/master/jakartaee-certification/5.21.2/tck-results-full-5.21.2.html
```

Comment:
The URL should be now like this:
```suggestion
https://docs.payara.fish/enterprise/docs/5.21.0/jakartaee-certification/5.21.2/tck-results-full-5.21.2.html
```
We have a redirect for old gitbook URLs for older versions but new versions should use the new Antora URLs. And the number after `docs/` is correctly 5.21.0. It's the version number of the documentation. We release new docs only for new minor Payara releases, not for patch releases. Patch releases only modify the docs for the previous minor version. I know, I know, it's not easy to get this right :-/